spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/event/BeforeConvertEvent.java

```diff
@@ -23,7 +23,7 @@
  * @since 1.1
  * @author Jens Schauder
  */
-public class BeforeConvertEvent<E> extends RelationalSaveEvent<E> {
+public class BeforeConvertEvent<E> extends RelationalEventWithEntity<E> {
```

Comment:
Changing the type hierarchy, especially when targeting backports is asking for trouble. The initial design of this class was already problematic as we never had an `AggregateChange` in the first place.